### ISSUE TITLE
Easy getter methods for the file meta table

### ISSUE THREAD

**Issue by Enet4** [Enet4/dicom-rs#70]
The attributes in the struct `dicom_object::meta::FileMetaTable` are always padded for even length when built via the builder. While this makes it easy to serialize, it is not very useful when inspecting the atrributes.

This calls for new methods to `FileMetaTable` which automatically trim the padding character out before returning the string slice.

Example signature (should be one per textual attribute):

```rust
fn transfer_syntax(&self) -> &str { unimplemented!() }
```

**Comment by PierreBou91** [comment]
Hi @Enet4 ,
I was browsing the `good first issues` to try to improve my Rust and dicom understanding when I saw this one.
When I went to the [FileMetaTable](https://github.com/Enet4/dicom-rs/blob/26ca8c2a4ea63f91f6cd2f9b53f7c9f23234cc81/object/src/meta.rs#L178) methods, I could see that you added the getter in [this commit](https://github.com/Enet4/dicom-rs/commit/42c0ea96f61779c01006122e9ed9d6317e3d193c#diff-f292e1aa4dcfbce114716ec39609bc940e4d85bbd871fd2f40d58e79bb5ef9de) and that it was merged.

So my question is the following, is it just that this issue should be closed or is there some more work I could help with ? If it's the case, could you provide a bit of guidance as to where I should start to handle this issue ?

Thanks a lot,

Pierre

**Comment by Enet4** [comment]
@PierreBou91 You're right, I ended up adding these helper methods over time. Thanks for the heads up.

There are a few other [issues with the good first issue label](https://github.com/Enet4/dicom-rs/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22), such as #46, but if you are interested in tackling some other task recorded here, feel free to let me know through that issue or reach out to me on [Zulip](https://dicom-rs.zulipchat.com/).
